.travis.yml

```diff
@@ -84,10 +84,12 @@ script:
   - blt deploy:build
   # Add Drupal VM config to repo.
   - blt vm:init
+  # Run the 'blt-project' phpunit tests.
+  - phpunit vendor/acquia/blt/tests/phpunit --group blt-project
```

Comment:
Don't these get executed already by the ci:build:validate:test target run in blt-project?